dts/bindings/dsa/microchip_dsa.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2020 DENX Software Engineering GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    DSA Device
+
+include: [spi-device.yaml]
+properties:
+    dsa-master-port:
```

Comment:
Suggest rewording 'master' and 'slave' throughout to align with Zephyr's Coding Guidelines
https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language

Reply:
I agree w/ @jukkar's comment. The coding guidelines specifically allow exemption for our usecase: "Exceptions are allowed for maintaining existing implementations or adding new implementations of industry standard specifications governed externally to the Zephyr Project.". We can rename the properties when they're changed in Linux.